core/templates/pages/about-page/about-page.component.html

```diff
@@ -1,261 +1,670 @@
+    height: auto;
+    padding-bottom: 5vh;
+    width: 100%;
+  }
+  .oppia-about-page-section-two .oppia-section-two-content {
+    align-items: center;
+    display: flex;
+    flex-direction: column;
+  }
+  .oppia-about-page-section-two .oppia-about-title {
+    padding-top: 6.5vh;
     text-align: center;
   }
-  .oppia-about-page-container .oppia-about-three-cols-layout {
-    display: inline-block;
-    margin-right: 20px;
-    width: -webkit-calc((100% - 60px) / 3);
-    width: -moz-calc((100% - 60px) / 3);
-    width: -o-calc((100% - 60px) / 3);
-    width: calc((100% - 60px) / 3);
+  .about-page .oppia-about-page-title {
+    color: #00424A;
+    font-family: "Capriola", Arial, sans-serif;
+    font-size: 40px;
+    font-weight: normal;
+    margin: 0;
+    width: 30vw;
   }
-  .oppia-about-page-container .oppia-about-three-cols-layout:last-of-type {
-    margin-right: 0;
+  .oppia-about-page-section-two .oppia-about-content {
+    text-align: center;
+    width: 768px;
   }
-  @media only screen and (max-width: 500px) {
-    .oppia-about-page-container .oppia-about-three-cols-layout {
-      width: 100%;
-    }
+  .about-page .oppia-about-page-content {
+    color: #333;
+    font-family: "Roboto", Arial, sans-serif;
+    font-size: 20px;
+    line-height: 1.5;
+    margin: 0;
+    padding-bottom: 5vh;
+    padding-top: 4vh;
+    width: 35vw;
   }
-  .oppia-about-page-container .oppia-about-three-cols-feature,
-  .oppia-about-page-container .oppia-about-three-cols-feature p {
+  .oppia-about-page-section-two .oppia-about-icons {
+    display: flex;
+    justify-content: center;
+    max-width: 1440px;
     text-align: center;
   }
-  .oppia-about-page-container .oppia-about-three-cols-feature,
-  .oppia-about-page-container .oppia-about-three-cols-feature h3 {
+  .oppia-about-page-section-two .oppia-about-features-container {
+    display: flex;
+  }
+  .oppia-about-page-section-two .oppia-about-feature-description-container {
+    width: 20%;
+  }
+  .oppia-about-page-section-two .oppia-about-feature-text {
+    padding: 2vh 4vw 0 4vw;
+  }
+  .oppia-about-page-section-three {
+    background-color: #00645C;
+    display: flex;
+    height: auto;
+    padding: 6.5vh 2vw 10vh 0;
+  }
+  .about-page .oppia-about-image-right {
+    float: right;
+    margin-left: auto;
+    margin-right: 2vw;
+    padding-left: 4vw;
     text-align: center;
+    width: 50%;
   }
-  .oppia-about-page-container .oppia-about-credits-letter-groups.three-col {
-    -webkit-column-count: 3;
-    -moz-column-count: 3;
-    column-count: 3;
-    -webkit-column-gap: 20px;
-    -moz-column-gap: 20px;
-    column-gap: 20px;
-    margin: 0 0 10% 0;
+  .about-page .oppia-about-content-left {
+    padding-left: 4vw;
   }
-  .oppia-about-page-container .oppia-about-credits-letter-groups span {
-    font-size: 1.8em;
+  .oppia-about-page-section-three .oppia-about-outcome-title,
+  .oppia-about-page-section-three .oppia-about-outcome-content {
```

Comment:
Define this as oppia-color-white and use it as a class, maybe?

Reply:
done